framework-cloud/framework-openshift/src/main/java/org/kie/cloud/openshift/scenario/WorkbenchWithKieServerScenarioImpl.java

```diff
@@ -129,18 +126,21 @@ public void deploy() {
             throw new RuntimeException("Malformed secure URL for kie server", e);
         }
 
-        logger.info("Waiting for Kie server deployment to become ready.");
-        kieServerDeployment.waitForScale();
-
-        logger.info("Waiting for Kie server to register itself to the Workbench.");
-        KieServerControllerClientProvider.waitForServerTemplateCreation(workbenchDeployment);
-
         databaseDeployment = new DatabaseDeploymentImpl();
         databaseDeployment.setOpenShiftController(openshiftController);
         databaseDeployment.setNamespace(projectName);
         databaseDeployment.setDatabaseName(DeploymentConstants.getDatabaseName());
         databaseDeployment.setApplicationName(OpenShiftConstants.getKieApplicationName());
 
+        logger.info("Waiting for Workbench deployment to become ready.");
+        workbenchDeployment.waitForScale();
+
+        logger.info("Waiting for Kie server deployment to become ready.");
+        kieServerDeployment.waitForScale();
```

Comment:
Database deployment should wait for scale before Kie server deployment. I think that this should be safer - Kie Server has database ready and can connect to it.